CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.5)
+project(eigen3_cmake_module)
+
+find_package(ament_cmake REQUIRED)
+
+if(BUILD_TESTING)
+  find_package(ament_cmake_lint_cmake)
+  find_package(ament_cmake_copyright)
```

Comment:
Find with `REQUIRED` - otherwise you can't assume the package was found and use it unconditionally.

Reply:
Require test dependencies when testing in 59474ecb715644efb9a20095e9b73350bbbd7bd7